tests/components/myuplink/conftest.py

```diff
@@ -167,3 +167,24 @@ async def init_integration(
     await hass.async_block_till_done()
 
     return mock_config_entry
+
+
+@pytest.fixture
+def platforms() -> list[str]:
+    """Fixture for platforms."""
+    return []
+
+
+@pytest.fixture
+async def setup_platform(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    platforms,
+) -> MockConfigEntry:
+    """Set up one or all platforms."""
```

Comment:
I had problems to set the returned type. This variant removed the complaints but I must admit that I don't relally understand if it is correct.